src/components/Contacts/ContactDetails/ContactDetailsTab/Tags/ContactTags.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactElement } from 'react';
+import { Box, CssBaseline, MuiThemeProvider } from '@material-ui/core';
+import { array } from '@storybook/addon-knobs';
+import theme from '../../../../../theme';
+import { ContactTags } from './ContactTags';
+
+export default {
+  title: 'Contacts/Tab/ContactDetailsTab/ContactTags',
+  component: ContactTags,
+};
+
+export const Default = (): ReactElement => {
+  return (
+    <Box m={2}>
+      <MuiThemeProvider theme={theme}>
+        <CssBaseline />
+        <ContactTags
+          contactId="contact_id"
+          contactTags={array('contactTags', ['help', 'something'])}
```

Comment:
Good idea to use the array addon here, it's neat to be able to edit the values dynamically.